distribution/docs/src/main/resources/_contents/_sources/federated-sources-contents.adoc

```diff
@@ -5,6 +5,8 @@ The following federated sources are included with a standard installation of ${b
 
 <<_csw_federated_source,CSW Federated Source>>:: Supports searching collections of descriptive information (metadata) for data, services, and related information objects.
 
+<<_csw_transactional_federated_source,CSW Transactional Federated Source>>:: Supports searching collections of descriptive information (metadata) for data, services, and related information objects as well as sending transactional queries to the remote source.
```

Comment:
"...as well as sending transactional queries to the remote source" is confusing
maybe "Additionally the transactional source supports CSW transactions including create, update, and delete operations to the remote source."